docs/root/configuration/advanced/well_known_filter_state.rst

```diff
@@ -56,6 +56,12 @@ The following list of filter state objects are consumed by Envoy extensions:
   <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.tunneling_config>` to disable tunneling on a
   per-connection bases. Accepts values "true" and "false".
 
+``envoy.filters.listener.local_replay_owner``
```

Comment:
`envoy.filters.network.http_connection_manager.local_reply_owner`

This filter state doesn't belong to one specific filter, so may this name is better.